plugins/CoreHome/vue/src/Comparisons/Comparisons.store.ts

```diff
@@ -292,6 +292,11 @@ export default class ComparisonsStore {
 
   private loadComparisonsDisabledFor() {
     const matomoModule: string = MatomoUrl.parsed.value.module as string;
+
+    if (typeof matomoModule === 'undefined') {
```

Comment:
In theory Matomo can also be opened without a module given in the URL when it's installed. In that case it will be handled as `CoreHome`. e.g. a URL like this should work: `index.php?action=index&idSite=1&period=day&date=yesterday#?period=day&date=yesterday&category=Dashboard_Dashboard&subcategory=1`

Maybe it's better to check if the current page is the installation page by checking the html element. e.g.
https://github.com/matomo-org/matomo/blob/680ac30b2078ce7f0fb587c145161dab0bce2c24/plugins/Installation/templates/layout.twig#L15